tsconfig.build.json

```diff
@@ -8,10 +8,10 @@
   "references": [
     // {"path": "modules/aggregation-layers"},
     // {"path": "modules/arcgis"},
-    // {"path": "modules/carto"},
+    {"path": "modules/carto"},
     {"path": "modules/core"},
     {"path": "modules/extensions"},
-    // {"path": "modules/geo-layers"},
+    {"path": "modules/geo-layers"},
```

Comment:
Are you sure you want to be uncommenting geo-layers in this PR?

Reply:
`geo-layers` are ready to be part of tsbuild, it's agreed with @Pessimistress